topics/icrc_28_trusted_origins.md

```diff
@@ -18,7 +18,9 @@
 This standard describes how a canister can indicate that a relying party (an entity that relies on the canister for certain functions or services) is trusted.
 
 Canisters that manage tradable assets or are otherwise meant to be composed upon by distinct parties in the ecosystem (e.g. ICRC-1 or ICRC-7 canisters), 
-**MUST** not implement ICRC-28: ICRC-28 privileges the listed entities to potentially act independently on behalf of the signer, which is a security risk in the context of tradable assets and shared infrastructure.
+**MUST NOT** implement ICRC-28: ICRC-28 privileges the listed entities to potentially act independently on behalf of the signer, which is a security risk in the context of tradable assets and shared infrastructure.
+
+> **Note:** If autonomy is required in the context of tradable assets it is recommended to use relying party delegations as per [ICRC-34](./icrc_34_delegation.md) in conjunction with [ICRC-2](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-2/README.md) or https://github.com/dfinity/ICRC/blob/main/ICRCs/ICRC-37/ICRC-37.md.
```

Comment:
What does it mean to require autonomy?

Reply:
I meant that an entity can act autonomously. I have changed the wording. Hopefully it is clearer now.